### PR TITLE
Gdi 95 add typing and comments in the fdp gdi repo

### PR DESCRIPTION
**Pull request by a-nayden**
- type hints added
- logging for other that 200 FDP response
- fix parse None issue and tests accordingly
- use rdflib namespaces instead of hard-coded predicates

### CODE REVIEW

File: ckanext/fairdatapoint/tests/test_fair_data_point.py [hunk 1]
```diff
@@ -70,13 +70,13 @@ def test_fdp_get_graph_pass_empty(self, mocker):
         assert fdp_get_data.call_count == 1
         assert to_isomorphic(actual) == to_isomorphic(Graph())
 
-    @pytest.mark.xfail(raises=ValueError)
+    # @pytest.mark.xpassed(raises=ValueError)
```
Comment (brunopacheco1) [inline ckanext/fairdatapoint/tests/test_fair_data_point.py:73]: Are you sure you want to keep this line commented?

**Comment by brunopacheco1** [comment]
@a-nayden excelent PR, thanks a lot for all those changes.

File: ckanext/fairdatapoint/tests/test_fair_data_point.py (hunk 1 above)
Comment (a-nayden) [reply ckanext/fairdatapoint/tests/test_fair_data_point.py:73]: sorry, I forgot to delete it

**Review by kburger — APPROVED** [review]
Looks like a great improvement. There are some minor things to be gained, like fstrings vs `.format` and shorthand `requests` calls, but that's for later.

**Pull request merged**